sefaria/model/text.py

```diff
@@ -1831,9 +1831,9 @@ def find_string(self, regex_str, cleaner=lambda x: x, strict=True):
         text_list = [x for x in self.ja().flatten_to_array() if len(x) > 0]
         if len(text_list) != len(ref_list):
             if strict:
-                raise ValueError("The number of refs doesn't match the number of starting words. len(refs)={} len(inds)={}".format(len(ref_list),len(ind_list)))
+                raise ValueError("The number of refs doesn't match the number of starting words. len(refs)={} len(inds)={}".format(len(ref_list),len(text_list)))
```

Comment:
This was a broken reference - at some point in time someone changed "ind_list" to "text_list" above but didn't change these references to it.